packages/components/inputs/localized-text-input/src/localized-text-input.styles.ts

```diff
@@ -25,6 +25,15 @@ const getLanguageLabelBackgroundColor = (props: TLocalizedInputProps) => {
   return designTokens.backgroundColorForLocalizedInputLabel;
 };
 
+const getLanguageLabelBorderColor = (props: TLocalizedInputProps) => {
+  if (props.isDisabled) {
+    return designTokens.borderColorForInputWhenDisabled;
+  }
+  return props.isReadOnly
```

Comment:
In the previous implementation we had the `isNewTheme` prop as part of the condition. Did you check we don't actually need it?

Reply:
Thanks Carlos, 
we don't need it as we're now using different design tokens.